ckan/logic/__init__.py

```diff
@@ -312,6 +313,15 @@ def clear_actions_cache():
     _actions.clear()
 
 
+def chained_action(func):
+    func.__is_chained_action__ = True
```

Comment:
`__double_underscore__` should be reserved for attributes that have a special meaning to the python interpreter. You can use `func.chained_action = True` here instead.